executor/analyze_test.go

```diff
@@ -759,3 +760,26 @@ func (s *testSuite1) TestDefaultValForAnalyze(c *C) {
 	tk.MustQuery("explain select * from t where a = 1").Check(testkit.Rows("IndexReader_6 1.00 root  index:IndexRangeScan_5",
 		"└─IndexRangeScan_5 1.00 cop[tikv] table:t, index:a(a) range:[1,1], keep order:false"))
 }
+
+func (s *testSuite1) TestIssue20874(c *C) {
```

Comment:
Better to use the scenario as func name, or when we want to sort out all test cases(see the coverage or optimize ci), it is hard to do such works.